functions/src/expressRestApp/routes/route.service.ts

```diff
@@ -0,0 +1,18 @@
+import express from "express";
+import { getData } from "../services/stations.service";
+
+const router = express.Router();
+
+router.get("/month", async (req, res) => {
+  await getData(req, res, "month");
+});
+
+router.get("/day", async (req, res) => {
+  await getData(req, res, "day");
+});
+
+router.get("/settings", async (req, res) => {
+  await getData(req, res, "settings");
+});

```

Comment:
we may update this route as above with /:stationId/settings or just use /:stationId in url to provide base info about particular station:
`router.get('/:stationId', async (req, res, next) => { try { const stationId = req.params.stationId; const stationData = await getData(req, res, stationId, "settings"); // getData should be updated with new argument stationId   res.status(200).json(stationData); } catch (error) { next(error); // Pass the error to the errorHandler middleware } });`